src/odemis/acq/stream/_projection.py

```diff
@@ -115,6 +115,13 @@ def __init__(self, stream):
         self.stream.auto_bc.subscribe(self._onAutoBC)
         self.stream.auto_bc_outliers.subscribe(self._onOutliers)
 
+        self.emitter = None
```

Comment:
All this shouldn't be necessary. That was a workaround for calling .addStream() on the streambar_controller, but that's not done anymore.